packages/account/src/test-utils/launchNode.ts

```diff
@@ -147,12 +145,11 @@ export const launchNode = async ({
   ip,
   port,
   args = [],
-  fuelCorePath = process.env.FUEL_CORE_PATH ?? undefined,
+  fuelCorePath = process.env.FUEL_CORE_PATH || undefined,
   loggingEnabled = true,
-  debugEnabled = false,
```

Comment:
I removed `debugEnabled` because all the messages of `fuel-core` are being piped into `stderr`. I also took a look in the https://github.com/fuellabs/fuel-core source code and found piping into `stdout` (which was used with `debugEnabled`) only for their `fuel-core-keygen` utility. @arboleya Did you add this? Or maybe @Dhaiwat10? If yes, can you provide example usage of it?

Reply:
I think the `loggingEnabled` suffice; we just want a way of turning logs on/off, I guess.